keyboards/kyria/keymaps/gotham/README.md

```diff
@@ -0,0 +1,49 @@
+#Gotham's Keymap for [Kyria](https://github.com/splitkb/kyria)
+
+## Keymap
+This is my personal keymap for Kyria with some mods.
+
+More information about the Kyria keyboard can be found [here](https://blog.splitkb.com/blog/introducing-the-kyria)
+
+### Rotary Encoders
+
+Press the encoder on each half to cycle between:
+- Volume
+- Word Nav (Ctrl + Left / Right)
+- Left / Right
+- Up / Down
+- Page Up / Page Down
+
+### OLEDs
+
+Master-side OLED displays dynamic data:
+- Current layer
+- Current mode of each rotary encoder
+
+Slave-side OLED currently only displays a static logo. More changes to come when I figure out how to drive two oleds with serial link.
+
+### Joystick
+
+A PSP 2000 joystick is attached to the right half. It will currently only function when the USB cable is connected to the right half. When I figure out how to transfer data between halves using serial link, I will make this work regardless of which side is the master.
```

Comment:
If you set `SPLIT_TRANSPORT = custom` and add your own transport file, you can send the report to the master side. 

If you message me on discord or what not, I can help with this.  

You can see what I did with the host LEDs here:
https://github.com/qmk/qmk_firmware/compare/master...drashna:feature/split_host_sync

Reply:
I'll take you up on that soon, thank you!